src/js/lib/monster.ui.js

```diff
@@ -15,7 +15,10 @@ define(function(require) {
 		footable = require('footable'),
 		mousetrap = require('mousetrap'),
 		Drop = require('drop'),
-		Clipboard = require('clipboard');
+		Clipboard = require('clipboard'),
+		moment = require('moment');
```

Comment:
You only need to require `moment-timezone`, as it [already depends on `moment`](https://github.com/2600hz/monster-ui/blob/master/src/js/main.js#L87).

```suggestion
		moment = require('moment-timezone');
```

Reply:
That's how it should work but since requirejs is not setup properly, we need to require each libs.